src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-table" context="ddl">
+        <createTable tableName="temporary_certificate">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="temporary_certificate_pkey"/>
+            </column>
+            <column name="certificate_number" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="gss_code" type="varchar(80)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="source_type" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="source_reference" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="application_reference" type="varchar(255)"/>
+            <column name="certificate_template_version" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issuing_authority" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="english_ero_id" type="uuid">
+                <constraints nullable="false"
+                             foreignKeyName="fk_temporary_certificate_english_ero_id"
+                             references="electoral_registration_office(id)"/>
+            </column>
+            <column name="issuing_authority_cy" type="varchar(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="welsh_ero_id" type="uuid">
+                <constraints nullable="true"
+                             foreignKeyName="fk_temporary_certificate_welsh_ero_id"
+                             references="electoral_registration_office(id)"/>
+            </column>
+
+            <column name="first_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="middle_names" type="varchar(255)"/>
```

Comment:
The current layout for the temp cert does not include the middle name(s), but we collect it for the main cert and it is sent to the print provider ... and presumably they print it on the cert if its provided?
I'm thinking we collect it in the table just in case they change the design in this respect!

Reply:
Even though there is only a `Name` on the temporary certificate - agree better to store the components separately.